tests/unit/resources/sample.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    service: <name>
```

Comment:
Why do we have angular brackets here? I.e., why `<name>` ?

Reply:
This is a sample file for check whether the implemented `replaceAll` function works as expected

Reply:
Used only for unit tests

Reply:
Ok